docs/HowTo/topics/Topic-overview.md

```diff
@@ -0,0 +1,50 @@
+# Topic Overview
+
+Topic overview provides different views of topic. 
+
+## Overview
+
+In this tab, the below are displayed
+
+- topic configuration like partitions, replication factor and if any advanced configuration
```

Comment:
Topic configuration information  including partition and replication factor details and if all advanced configuration is viewable here.